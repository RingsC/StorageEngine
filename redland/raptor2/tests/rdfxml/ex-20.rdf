<?xml version="1.0"?>
<rdf:RDF xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
	 xmlns:a="http://example.com/properties/">
  <!-- test that rdf:ID generates the right URI -->
  <rdf:Description rdf:ID="foo">
     <a:property rdf:resource="http://example.org/object"/>
  </rdf:Description>
</rdf:RDF>
