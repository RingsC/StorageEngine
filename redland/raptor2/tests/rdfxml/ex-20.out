<http://librdf.org/raptor/tests/ex-20.rdf#foo> <http://example.com/properties/property> <http://example.org/object> .
